qf_lib/analysis/tearsheets/strategy_monitoring_document.py

```diff
@@ -81,15 +81,27 @@ def set_in_sample_excess_statistics(self, excess_is_mean_return, excess_is_sigma
         self.excess_is_mean_return = excess_is_mean_return
         self.excess_is_sigma = excess_is_sigma
 
-    def _add_excess_cone_chart(self):
-        diff = self.strategy_series.to_simple_returns().subtract(self.benchmark_series.to_simple_returns(),
-                                                                 fill_value=0)
+    def _add_cone_chart(self):
+        if self.is_sigma and self.is_mean_return:
+            diff = self.strategy_series.to_simple_returns()
+            title = "Returns"
+            mean_return = self.is_mean_return
+            sigma = self.is_sigma
+        elif self.excess_is_sigma and self.excess_is_mean_return:
+            diff = self.strategy_series.to_simple_returns().subtract(self.benchmark_series.to_simple_returns(),
+                                                                     fill_value=0)
+            title = "Excess returns"
+            mean_return = self.excess_is_mean_return
+            sigma = self.excess_is_sigma
+        else:
+            raise ValueError("You need to set stats for either returns or excess returns")
```

Comment:
Let's mention that one needs to set parameters (not stats) to create this report using either ... or ... function (so that the user knows how to set things)